src/point.rs

```diff
@@ -10,8 +10,12 @@
 use super::UnknownUnit;
 use approxeq::ApproxEq;
 use length::Length;
+#[cfg(feature = "mint")]
+use mint;
```

Comment:
Another duplication (I assume git merge hazard?). Most likely the reason travis is upset.